inference.Rmd

```diff
@@ -328,7 +342,7 @@ population_parameters
 ```
 The price per night of all Airbnb rentals in Vancouver, BC 
 is \$`r round(population_parameters$pop_mean,2)`, on average. This value is our
-**population parameter** since we are calculating it using the population data. 
+**population parameter** since we are calculating it using the population data. \index{population!parameter}
```

Comment:
We should probably remove the bolding here since we define it in the first paragraph of this chapter already

Reply:
fixed